emojivoto/emojivoto.yml

```diff
@@ -132,3 +126,27 @@ spec:
   - name: http
     port: 80
     targetPort: 80
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: vote-bot
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: vote-bot
+  template:
+    metadata:
+      labels:
+        app: vote-bot
+    spec:
+      containers:
+      - name: vote-bot
+        image: buoyantio/emojivoto-web:v2
+        command: ["emojivoto-vote-bot"]
+        env:
+        - name: WEB_HOST
+          value: "web-svc.emojivoto.svc.cluster.local:80"
```

Comment:
If you move this to the emojivoto namespace, then you can use "web-svc:80" as the web host, and that will exercise more of the destination api.